docs/it/come-affrontare-l-eta-nel-mondo-it.md

```diff
@@ -0,0 +1,76 @@
+
+La carriera di chi sviluppa è complessa da molti punti di vista e non priva di incognite e, al primo posto, possiamo mettere il fatto che sia una categoria nuova e in continua evoluzione. Basti guardare ad alcuni schemi che rappresentano la crescita e le conoscenze di una certa mansione che ci troviamo davanti ad un albero di scelte impressionante. Cercando di allontanarsi un poco per vedere un percorso di massima non possiamo non pensare alla crescita verticale verso mansioni mano a mano più gestionali e meno operative, è un pensiero che prima o poi si tende a fare. Questo è legato non solo all’ambizione ma anche ad un pensiero sul proprio futuro, in generale sull’invecchiamento intellettuale e sulle conseguenze che inevitabilmente può avere sul nostro rendimento, sulla nostra freschezza e sulla capacità di stare dietro ad un mondo, quello informatico, in perenne vorticoso movimento; questo rendimento è legato a doppio filo con il nostro intelletto.
+
+Dal punto di vista della catalogazione l’intelligenza, come capacità di risolvere un problema, viene descritta come Fluida o Cristallizzata (teorizzata dallo psicologo Cattell). La prima, quella fluida, è il pensiero logico, la nostra capacità di risolvere problemi nuovi. Quella cristallizzata, invece, è la nostra capacità di utilizzare le conoscenze pregresse; questa è quella che di solito cade sotto il nome di “seniority”. Nel lavoro dello sviluppatore sono importanti entrambe probabilmente in egual misura.
+
+Il passare del tempo va intuitivamente beneficio dell'intelligenza cristallizzata perché ci da un bacino più ampio a cui attingere. In passato questa catalogazione veniva legata al Q.I. da cui prende anche il nome: era il rapporto tra età psicologica e anagrafica (da cui quoziente). L’intelligenza fluida, invece, ha una caratteristica inversa, cioè con l’età ha una tendenza a calare, facendoci perdere freschezza. Le nostre performance come velocità, capacità di visualizzazione ed elaborazione sono legate a questi due elementi che sommati formano il nostro intelletto.
+
+Da questo punto di vista non stiamo dicendo nulla di nuovo perché è un sentire comune ed è da qui che a volte emerge la volontà di avanzare nella carriera allontanandosi dalla parte operativa che è considerata più avida del primo tipo di intelligenza. Con l’avanzare delle decadi, ad ogni passo, si fa sempre più pressante questa preoccupazione, ma la domanda che ci dobbiamo porre è quanto questa discesa incida sul serio e quanto, invece, possa essere culturale.
+
+Gli studi più recenti a riguardo sono stati svolti dall’Università di Monaco, racchiusi in un articolo dal titolo ["Life cycle patterns of cognitive performance over the long run" di Anthony Strittmatter](https://www.pnas.org/doi/10.1073/pnas.2006653117) che ha studiato e cercato di dare una risposta quantitativa e non solo qualitativa alla domanda: _quanto è differente un cervello giovane da uno vecchio e quale è il periodo in cui rende di più?_.
```

Comment:
Aggiungerei un ndr dove dici l'anno in cui è stata scritta questa cosa. Adesso sono gli ultimi studi fatti ma chissà quando qualcuno leggerà questo capitolo (magari motiva anche qualcuno a cercare qualcosa di più recente se vuole contribuire)

Reply:
Mi riferisco alla frase:

> Gli studi più recenti [...]